core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/SequencerPartitionContainerImpl.java

```diff
@@ -46,9 +53,15 @@
  * @date 14-May-2012
  * @since 0.1.6
  */
+@Entity
+@Table(name = "SequencerPartitionContainer")
 public class SequencerPartitionContainerImpl extends AbstractSequencerPartitionContainer<SequencerPoolPartition> implements Serializable {
   protected static final Logger log = LoggerFactory.getLogger(SequencerPartitionContainerImpl.class);
 
+  @OneToMany(targetEntity = PartitionImpl.class, cascade = CascadeType.ALL)
+  @JoinTable(name = "SequencerPartitionContainer_Partition", joinColumns = {
+      @JoinColumn(name = "container_containerId") }, inverseJoinColumns = {
+          @JoinColumn(name = "partitions_partitionId") })
   private List<SequencerPoolPartition> partitions = new AutoPopulatingList<SequencerPoolPartition>(PartitionImpl.class);
```

Comment:
I'm concerned about how the assumptions of having an `AutoPopulatingList` are going to work when this is reloaded from the database and now a Hibernate list. Maybe make a note in the testing bug?